client/packages/invoices/src/Prescriptions/api/hooks/usePrescription.ts

```diff
@@ -174,7 +138,10 @@ const useGetById = (invoiceId: string | undefined) => {
 
     if (invoice?.__typename === 'InvoiceNode') {
       return invoice;
+    } else {
+      throw new Error('Could not find invoice');
     }
+    // TODO check if this is throwing unweildy errors where program component has id for yet to exist presciptions.
     // Don't throw error for this one if not found -- it's mainly used by
     // Program component (Prescription), which may have stored an id for a
     // prescription that doesn't yet exist
```

Comment:
I wasn't sure if this was a common case where prescriptions would have a stored id. Should we be throwing errors here? Or withholding errors?

Another option to make a prescription specific api which doesn't throw errors if not found. This could help if stored IDs is a common occurrence for prescriptions.